### ISSUE TITLE
aspect ratio problem

### ISSUE THREAD

**Issue by Cavencruiser** [doomhack/GBADoom#50]
Is it possible for you to add an option for corrected aspect ratio on sprites, not only to correct the squashed shape of everything but also extend the viewing field to take advantage of the gba's wider screen instead of rendering the pixels 1:1?

**Comment by Cavencruiser** [comment]
![Screenshot (140)](https://github.com/doomhack/GBADoom/assets/93107023/e7ba770d-868f-4eed-adc9-2bcf9c6bd981)
![Screenshot (141)](https://github.com/doomhack/GBADoom/assets/93107023/6c493f56-8527-4a6c-8216-39bab8985521)

**Comment by Kippykip** [comment]
I have a feeling it involve multiplying the height with a decimal value, which would might add an unnecessary strain on the already weak CPU doing that to every sprite drawn to be honest.

**Comment by Cavencruiser** [comment]
Yeah, I noticed in the source code that the game uses the original pc resolution as a base and then resizes it to the gba screen resolution from what I read
